locust/main.py

```diff
@@ -589,7 +594,8 @@ def start_automatic_run():
         elif not options.worker and not environment.shape_class:
             logger.info("No run time limit set, use CTRL+C to interrupt")
 
-    if options.csv_prefix:
+    # `stats_csv_writer` is initialized with a StatsCSVFileWriter object if options.csv_prefix is "True"
```

Comment:
type hinting this makes it clear how confusing it is. maybe add a comment that it should be fixed some time?